### PR TITLE
BAM-566 pixel implementation walkthrough

### PR DESCRIPTION
**Pull request by PhilWinchester**

### CODE REVIEW

File: docs/tagmerchant.rst [hunk 1]
```diff
@@ -192,5 +192,81 @@ information into our data layer using Javascript before loading the Narrativ tag
...
+
+- Add box under "Triggering" to add a trigger for this tag.
+
+.. image:: _static/pixel_implementation_screenshots/checkout_4_add_trigger.png
+
+- If you don’t already have a trigger for checkout pages, then it’s easy to create one!
+
+.. image:: _static/pixel_implementation_screenshots/checkout_5_add_confirmation_page_trigger.png
+
+- The above example has a series of triggers based off of common URL types for checkout or confirmation pages. (E.g. narrative.com/checkout/ will trigger the “Checkout Page” option). Replace “checkout” with whatever word your site uses on checkout pages. Remember, you only need one trigger!
```
Comment (csdev) [inline docs/tagmerchant.rst]: `narrativ.com/checkout/`

File: docs/tagmerchant.rst [hunk 2]
```diff
@@ -192,5 +192,81 @@ information into our data layer using Javascript before loading the Narrativ tag
...
+
+- Select "Custom HTML" to open a text field to copy and paste the above code. Make sure you replace "ACCOUNT NAME" with your Narrativ account name.
+
+.. image:: _static/pixel_implementation_screenshots/2_choose_custom_html.png
+
+- Enter the code from the "Page View Tag" snippet (See the section before this). Replace the content of "BAMX_EVENT_DATA" with the relevant data from your data layer.
+
+.. image:: _static/pixel_implementation_screenshots/3_enter_tag_html.png
+
+- Select “All Pages” as the correct trigger for this "Page View Tag."
```
Comment (csdev) [inline docs/tagmerchant.rst]: don't use smart quotes in rst code

File: docs/tagmerchant.rst [hunk 3]
```diff
@@ -192,5 +192,81 @@ information into our data layer using Javascript before loading the Narrativ tag
...
+
+- Preview the changes made and fix any errors that pop up in the window.
+
+.. image:: _static/pixel_implementation_screenshots/submit_2_review.png
+
+- Click the “Submit” button to save your changes (this step is not final)
+
+.. image:: _static/pixel_implementation_screenshots/submit_3_submit_changes.png
+
+- Double check that everything you worked on is in this submission. Name the submission something like “Adding Narrative Pixel” so that it’s easy to find if you need to go back and debug any issues in the future.
```
Comment (csdev) [inline docs/tagmerchant.rst]: "Narrativ Pixel"

File: docs/tagmerchant.rst [hunk 4]
```diff
@@ -173,24 +203,100 @@ information into our data layer using Javascript before loading the Narrativ tag
...
+              a.parentNode.insertBefore(b, a);
+          } catch (e) {}
+      }("ACCOUNT NAME"));
+  </script>
+  <!-- end NARRATIV jstag -->
+
+Google Tag Manager Walkthrough
+------------
+
+Below is a walkthrough for implementing the Narrativ pixel using a "Custom HTML" tag in Google Tag Manger.
```
Comment (csdev) [inline docs/tagmerchant.rst]: "Manger"

Comment (PhilWinchester) [reply docs/tagmerchant.rst]: Nw branding. W drop th 'e' from vrything

**Pull request merged**